virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc

```diff
@@ -24,48 +32,32 @@ The network attachment definition must be in the same namespace as the pod or vi
 
 [NOTE]
 ====
-There are multiple methods for configuring a VLAN, including network attachment definition and node network configuration policy.
-However, a network attachment definition provides a more efficient and more manageable configuration.
+There are multiple methods for configuring a VLAN, for example, creating a node network configuration policy. However, a network attachment definition provides a more efficient and manageable configuration
 ====
 
-include::modules/virt-networking-glossary.adoc[leveloffset=+1]
-
-[id="virt-creating-linux-bridge"]
-== Configuring a Linux bridge
-
-include::modules/virt-creating-linux-bridge-nncp.adoc[leveloffset=+2]
+include::modules/virt-creating-linux-bridge-nncp.adoc[leveloffset=+1]
 
 For more information about scheduling, interface types, and other node networking activities,
 see the xref:../../../virt/node_network/virt-updating-node-network-config.adoc#virt-about-nmstate_virt-updating-node-network-config[node networking]
```

Comment:
You should remove this sentence, first because it's not necessary and second because the link will be obsolete by 4.12.

Reply:
Removed.